### PR TITLE
Clarifying comments in images sample

### PR DESCRIPTION
**Pull request by dzenanz**

### CODE REVIEW

**Comment by rodburns** [comment]
Thanks for your suggested changes. We are discussing amongst the team whether this needs further clarification within the comments. I'll update you on that.

File: samples/images.cpp [hunk 1]
```diff
@@ -88,7 +88,7 @@ int main() {
         auto coords = int2(item[0], item[1]);
 
         /* In SYCL images are read using a subscript operator to provide the
-         * coordinates, with an optional function call operator preceding it
+         * coordinates, with an optional function call operator following it
          * to provide a sampler. */
```
Comment (DuncanMcBain) [inline samples/images.cpp]:
```suggestion
        /* In SYCL images are read using a coordinates and sampler
         * which can interpolate image data between pixels.
         */
```

**Comment by DuncanMcBain** [comment]
I'm sorry it's taken so long for me to look at your PR @dzenanz, I've made a suggestion since it looks like the comment is very out-of-date with respect to the code (a previous sampler API had a much different look). Thank you for your contribution, if you apply this suggestion I'll get it merged!

**Comment by dzenanz** [comment]
I made a force-push to [my fork](https://github.com/dzenanz/computecpp-sdk), but this PR does not reflect it yet.

**Comment by DuncanMcBain** [comment]
Thank you very much!

**Pull request merged**